helm/vertical-pod-autoscaler-app/values.yaml

```diff
@@ -68,10 +71,15 @@ updater:
   # updater.enabled -- If true, the updater component will be deployed
   enabled: true
   # updater.extraArgs -- A key-value map of flags to pass to the updater
-  extraArgs: {}
+  extraArgs:
+    # update.extraArgs.min-replicas -- Minimum number of replicas to perform update
+    min-replicas: 1
+    # update.extraArgs.v -- Verbosity level for updater
+    v: 2
```

Comment:
This is the setting which we extract from config repo, IMHO it makes sense. We can drop the config repo setting afterwards.